MsmqReturnToSourceQueue/Program.cs

```diff
@@ -42,7 +46,7 @@ static void Main(string[] args)
         {
             Console.WriteLine("Input queue [{0}] resides on a remote machine: [{1}].", errorQueueAddress.Queue, errorQueueAddress.Machine);
             Console.WriteLine("Due to networking load, it is advised to refrain from using ReturnToSourceQueue on a remote error queue, unless the error queue resides on a clustered machine.");
-            if (!script)
+            if (!script && !skipLocalCheck)
```

Comment:
why is this needed?

Reply:
The ability to skip the prompt and ReadKey() allows us to preclude user input.

Reply:
isn't that what the `script` var is for?

Reply:
oh, nevermind, I see now that you added this so that we could force it to prompt, even if `inputQueue` is provided

Reply:
Oh, you're right. I missed that script was true by default.